test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -375,31 +378,60 @@ static void test_out_in_common(ipsec_test_flags *flags,
 		},
 	};
 
+	test.flags = *flags;
+
+	if (flags->stats == IPSEC_TEST_STATS_PROTO_ERR)
+		test.in[0].status.error.proto = 1;
+
+	if (flags->stats == IPSEC_TEST_STATS_SUCCESS) {
+		CU_ASSERT_EQUAL(odp_ipsec_stats(sa_out, &stats), 0);
+		CU_ASSERT_EQUAL(stats.success, 0);
+		CU_ASSERT_EQUAL(odp_ipsec_stats(sa_in, &stats), 0);
+		CU_ASSERT_EQUAL(stats.success, 0);
+	}
```

Comment:
Why not check here the initial values of all counters, not just the success counter, and in all cases, not just when testing the success counter?

Reply:
Agreed.